src/ComboBox/index.jsx

```diff
@@ -477,6 +495,8 @@ export default class ComboBox extends Component
             }
         }
 
+        const options = normalizeOptions( this.props.options ) ||
```

Comment:
`getDerivedStateFromProps()` will set the normalized options as `this.state.options`, then it wouldn't be necessary to call this function here again